### PR TITLE
elektroid: init at 2.1

### PR DESCRIPTION
**Pull request by dag-h**
###### Description of changes

[Elektroid](https://github.com/dagargo/elektroid) is a management application for Elektron devices.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/elektroid/default.nix [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "elektroid";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "dagargo";
+    repo = pname;
+    rev = "${version}";
```
Comment (bobby285271) [inline pkgs/applications/misc/elektroid/default.nix:29]:
```suggestion
    rev = version;
```

File: pkgs/applications/misc/elektroid/default.nix [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  buildInputs = [
+    wrapGAppsHook
+    autoreconfHook
+    gtk3
+    pkg-config
+    gettext
+    json-glib
+    automake
+    autoconf
+    libtool
```
Comment (bobby285271) [inline pkgs/applications/misc/elektroid/default.nix:42]: Should some of these go to nativeBuildInputs? https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/build-tools-in-build-inputs.md

File: pkgs/applications/misc/elektroid/default.nix [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    zlib
+    libjson
+    libzip
+  ];
+
+  meta = with lib; {
+    description = "Transfer application for Elektron devices";
+    homepage = "https://github.com/dagargo/elektroid";
+    maintainers = with maintainers; [ dag-h ];
+    license = licenses.gpl3;
```
Comment (bobby285271) [inline pkgs/applications/misc/elektroid/default.nix:56]:
```suggestion
    license = licenses.gpl3Plus;
```

Comment (dag-h) [reply pkgs/applications/misc/elektroid/default.nix]: Thanks for the review! I fixed up the requested changes. What do I do with the new commits? Should they be squashed into one "elektroid: init at 2.1" commit and force-pushed to my fork?

**Comment by camelpunch** [comment]
Please see my more recent submission for 2.5.2: https://github.com/NixOS/nixpkgs/pull/227964

**Comment by camelpunch** [comment]
This can now be closed as https://github.com/NixOS/nixpkgs/pull/260007 got merged.

**Pull request closed without merging**